### PR TITLE
Alena Zubritskaya-1

### PR DESCRIPTION
**Pull request by superpuper777**
## Name
Alena Zubritskaya

## Homework#
1
## Comment
Level 1.

## Checklist
- [ ] I have read homework assignment fully
- [ ] I have implemented homework assignment fully
- [ ] My pull request is named "#{name} #{surname} - #{homework_number}"
- [ ] My code is located under "#{name} #{surname}/#{homework_number}"
- [ ] It executes correctly
- [ ] It doesn't crash, freeze, kill everyone on unexpected input
- [ ] **Credentials are NOT hardcoded**
- [ ] **Large files and credentials added to .gitignore INSIDE homework dir**
- [ ] It can be tested in under 10 minutes (download part + parse + execute)
- [ ] Gemfile added, it runs under bundle exec without error

### CODE REVIEW

File: Alena Zubritskaya/1/lib/wordplay_fevr_bot.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
+require 'telegram/bot'
+require './database.rb'
+
+class MyBot
+  def initialize
+    @token = '603660924:AAFQSiTlEJNAh1zCt4MgwAkCZt1rprEebjc'
```
Comment (RGBD) [inline Alena Zubritskaya/1/lib/wordplay_fevr_bot.rb]: Credentials should not be hardcoded

File: Alena Zubritskaya/1/lib/database.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+require './parser'
+
+class Database
+  def initialize
+    @redis = Redis.new
+    @parser = Parser.new.getwordplay
+  end
+
+  def setdata
+    (0...@parser.length).each do |i|
```
Comment (RGBD) [inline Alena Zubritskaya/1/lib/database.rb]: https://ruby-doc.org/core-2.2.0/Integer.html#method-i-times

File: Alena Zubritskaya/1/lib/database.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+      @redis.set(i, @parser[i])
+    end
+  end
+
+  def getdata
+    i = rand(0...@parser.length)
```
Comment (RGBD) [inline Alena Zubritskaya/1/lib/database.rb]: rand(@parser.length)

**Review by RGBD — CHANGES_REQUESTED** [review]

**Comment by superpuper777** [comment]
What's mean invalid?

**Pull request closed without merging**